### ISSUE TITLE
Empty line before primary constructor is not reported and formated-out

### ISSUE THREAD

**Issue by marco89nish** [pinterest/ktlint#1004]
Version 0.39.0 (or earlier) introduced unwanted newline before annotated primary constructor (see #970). While that is fixed, ktlint format doesn't remove the extra line and ktlint check doesn't report it as an issue. IJ reformat code option removes the line by default but ktlint doesn't. 

## Expected Behavior
Given code: 
```
class A

constructor(a: Int)
```
check should report issue and format should remove empty line

## Observed Behavior
check doesn't report the issue and format doesn't change the file. 

## Steps to Reproduce
Given above

## Your Environment
* Version of ktlint used: 0.40.0
* Version of Gradle used (if applicable): 6.6.1

**Comment by marco89nish** [comment]
Looking into it a bit better, it has the same behavior with 
```
private class A

constructor(a: Int)
```
Maybe my .editorconfig file has some override. Does anyone know exact property for something like this?

**Comment by marco89nish** [comment]
It looks to be due to .editorconfig (large project), sorry for false reporting.
